### DISCUSSION TITLE
Clase Viernes 14/04

### DISCUSSION THREAD

**Discussion by martupiru** [harpomaxx/algoritmos2#17] [General]
Hola profe, buen día. Cómo está? 
Hemos estado hablando con mis compañeros y queriamos saber si existe la posibilidad de que la clase de hoy sea virtual?

**Comment by harpomaxx** [comment]
Hola, si la vamos a hacer virtual/hibrida. Por aca les voy a pasar la info.
Saludos

**Comment by gabriags** [comment]
Buenas tardes profes. No va a venir nadie a la facultad entonces? No vi ningun aviso de que hoy seria virtual

**Comment by harpomaxx** [comment]
Nunca es "virtual". Siemrpe es hibrida. Es decir, quien quiera venir a
clases presencial, va a poder hacerlo siempre. Yo vengo siempre. Hoy estuvo
Franco en el aula y el resto por meet.
